File: customview/src/main/java/com/hysea/customview/MainActivity.java
package com.hysea.customview;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }


    public void toPropertyView(View view) {
        Intent intent = new Intent(this, ViewPropertyActivity.class);
        startActivity(intent);
    }
}

File: animator/src/main/java/com/hysea/animator/MainActivity.kt
package com.hysea.animator

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import com.hysea.animator.propertyAnimator.PropertyAnimatorActivity
import com.hysea.animator.transformer.TransformerActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun toTransformer(view: View) {
        startActivity(Intent(this, TransformerActivity::class.java))
    }

    fun toPropertyAnimator(view: View) {
        startActivity(Intent(this, PropertyAnimatorActivity::class.java))
    }
}
